Models/User.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+///////////////////////////////
+// MODELS
+////////////////////////////////
+const VideoidSchema = new Schema(
+  {
+  URL: String,
+  log: String,
+  },
+    { timestamps: true }
+);
+
+const ObjectidSchema = new Schema(
```

Comment:
We should change the name of this Schema to make it a little more clear of what the data schema is. Lets change "const ObjectidSchema" to const "LogSchema"